Functions/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Can we use the existing `.gitignore` file instead of creating a new one?

Reply:
Done